packages/redux-api-params-middleware/src/middleware.js

```diff
@@ -6,16 +6,19 @@ import { parse, stringify } from 'qs';
 import type { Dispatch } from 'redux';
 import type { Action } from './types';
 
-function parametriseEndpoint(endpoint, params) {
+function parametriseEndpoint(endpoint, params, options = { arrayFormat: 'indices' }) {
```

Comment:
Please use `parametrize` instead. We had a discussion about using the American spelling of words.

Reply:
Oh sorry, it's not your fault. I just noticed that it was there before. You don't have to change that in this PR